stdlib/os/__init__.pyi

```diff
@@ -246,9 +248,13 @@ class _Environ(MutableMapping[AnyStr, AnyStr], Generic[AnyStr]):
     def __iter__(self) -> Iterator[AnyStr]: ...
     def __len__(self) -> int: ...
     if sys.version_info >= (3, 9):
+        def __class_getitem__(cls, __item: Any) -> GenericAlias: ...
```

Comment:
Why is this here? I don't see it in the implementation.

Reply:
You're right, it's inherited from `collections.abc.MutableMapping` -- I forgot that classmethods became bound to the class even if you accessed them from the class, so did `type(environ).__class_getitem__ is MutableMapping.__class_getitem__` as a test, which evaluated to `False`... etc. etc...

Reply:
Although we don't actually define it on `MutableMapping` in typeshed due to the fact that we pretend it's the same class as `typing.MutableMapping`... But that's a subject for another issue (and probably not really worth fussing about, anyway).